Makefile

```diff
@@ -57,23 +57,28 @@ verify-lint-fix:
 	make lint-fix 2>/dev/null || true
 	git diff --exit-code
 
-.PHONY: verify-scripts
-verify-scripts:
-	bash -x hack/verify-deepcopy.sh
-	bash -x hack/verify-openapi.sh
+# Verify codegen runs all verifiers in the order they are defined in the root.go file.
+# This includes all generators defined in update-codegen, but also the crd-schema-checker and crdify verifiers.
+.PHONY: verify-codegen
+verify-codegen:
+	EXTRA_ARGS=--verify hack/update-codegen.sh
+
+.PHONY: verify-non-codegen
+verify-non-codegen:
 	bash -x hack/verify-protobuf.sh
-	bash -x hack/verify-swagger-docs.sh
```

Comment:
I don't see swaggerdocsgenerator added to all verifiers in root.go. Is that an oversight, or is it included some other way?